packages/docs/content/docs/tutorials/supabase-mcp-server.mdx

```diff
@@ -0,0 +1,584 @@
+process.on("SIGINT", () => process.exit(0));
+process.on("SIGTERM", () => process.exit(0));
+
+console.error("✅ Supabase MCP server running (STDIO, NL → SQL enabled)");
+```
+
+#### Writing the `exec_sql` function
+
+In the `index.js` file, you’d notice the following code:
+```jsonc
+const { data, error } = await supabase.rpc("exec_sql", { sql_query });
+```
+
+This allows your MCP server to send queries to Supabase. When the server calls the code above, it sends your generated SQL string to the `exec_sql` function in Supabase, which executes it on the server and returns the results as JSON
+
+Now, Supabase does not support executing SQL directly through the client SDK. To make this work, we need to create an `exec_sql` RPC function in Supabase editor. Here’s the code you need to execute:
+```sql
+create or replace function exec_sql(sql text)
+returns setof json as $$
+begin
+  return query execute sql;
+end;
+$$ language plpgsql;
+```
+![Exectue RPC function](./images/exec-function-supabase-dashboard.png)
+
+### Step 4: Configure Pochi to add your MCP server config
+
+In your VS Code, press `Command + Shift + P` on Mac, or `Ctrl + Shift + P` on Windows and open the `Pochi: Open MCP Server Settings`. Here, you need to add your MCP server config. Initially it is empty: 
+```jsonc
+“mcp": {}
+```
+Paste this JSON snippet inside the existing MCP config :
+```jsonc
+"mcp": {
+    "my-supabase-mcp": {
+      "command": "/Users/apple/.nvm/versions/node/v20.19.0/bin/node",
+      "args": [
+        "index.js"
+      ],
+      "cwd": "/Users/apple/Documents/supabase-mcp-server",
+      "disabledTools": []
+    }
+  }
+```
+The above code snippet will register a local MCP server in Pochi.
+
+* `command` = This is the path to the Node.js executable you want to use.
+* `args` = The actual MCP server package or script to run (which we will generate in Step 6)
+* `cwd` = The current working directory where the command will run.
+* `disabledTools` = Allows you to disable certain tools or integrations (like linters or debuggers). It’s currently empty, so no tools are disabled.
+
+When you save this config in the `~/config.json` file, Pochi will know to launch `node index.js` as a subprocess and communicate with it via stdio transport when the “my-supabase-mcp” server is enabled. Read more about it **[here](https://docs.getpochi.com/mcp/)**.
+
+
+
+#### Enable your MCP server
+
+Next, we need to go to Pochi’s Tools Settings page (you can also find it via Command Palette: `Pochi: Open Tools Settings`)
+
+Look for the MCP server named `my-supabase-mcp` and toggle it ON (enabled). You should see a green dot to indicate its connected.
+
+<div
+  style={{
+    position: "relative",
+    paddingBottom: "53.7%",
+    marginBottom: 20,
+    height: 0,
+    overflow: "hidden",
+    maxWidth: "100%",
+  }}
+>
+  <iframe
+    src="https://drive.google.com/file/d/1txzxWQ5R0fRZVSme1UtCRb77WguU0O6H/preview"
+    style={{
+      position: "absolute",
+      top: 0,
```

Comment:
The  property isn't used in this tutorial. To simplify the configuration and make it easier for readers to follow, it would be best to remove it and its corresponding explanation.